tesseract_core/runtime/mpa.py

```diff
@@ -149,9 +149,9 @@ def __init__(self, base_dir: Optional[str] = None) -> None:
             # If it's a file URI, convert to local path
             tracking_uri = tracking_uri.replace("file://", "")
 
-            # Relative paths are resolved against the log_dir
+            # Relative paths are resolved against the base output path
             if not Path(tracking_uri).is_absolute():
-                tracking_uri = (Path(self.log_dir) / tracking_uri).resolve()
+                tracking_uri = (Path(get_config().output_path) / tracking_uri).resolve()
```

Comment:
```suggestion
                tracking_uri = (Path(base_dir) / tracking_uri).resolve()
```

Reply:
this will work for `tesseract run` but not for `tesseract serve`, in which case the behavior would be the same as it is now, right?

Reply:
Hmm yea you're right. This is a bit messy but ok for now.